### ISSUE TITLE
Package is using deprecated npm modules

### ISSUE THREAD

**Issue by targos** [NodeSecure/cli#122]
```
> npm install @nodesecure/cli -g
npm WARN deprecated readdir-scoped-modules@1.1.0: This functionality has been moved to @npmcli/fs
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
```

**Comment by fraxken** [comment]
Just updated the dependencies of the CLI, I also need to update scanner dependencies (npm published a lot of major recently). Anyway should be fixed next week.

**Comment by fraxken** [comment]
Fixed with v2.2.0
